tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -314,11 +352,25 @@ export class PodmanDesktopRunner {
       mkdirSync(parentDir, { recursive: true });
     }
 
+    console.log(`[DEBUG] stringifying settingsContent`);
     const settingsContent = JSON.stringify({
       'preferences.OpenDevTools': 'none',
       'extensions.autoCheckUpdates': this._autoCheckUpdate,
       'extensions.autoUpdate': this._autoUpdate,
+      'extensions.disabled': [
+        !this._composeEnabled ? 'podman-desktop.compose' : '',
+        !this._dockerEnabled ? 'podman-desktop.docker' : '',
+        !this._kindEnabled ? 'podman-desktop.kind' : '',
+        !this._kubeContextEnabled ? 'podman-desktop.kube-context' : '',
+        !this._kubectlCliEnabled ? 'podman-desktop.kubectl-cli' : '',
+        !this._registriesEnabled ? 'podman-desktop.registries' : '',
+        !this._limaEnabled ? 'podman-desktop.lima' : '',
+      ],
+      'podman.binary.path': !this._isLinux
+        ? `${process.env.USERPROFILE}\\tools\\podman\\podman-5.2.0\\usr\\bin\\podman.exe`
+        : '/usr/bin/podman',
```

Comment:
Why are we hardcoding some paths here? What if the test is running on mac? Why is this at all?

Reply:
yeah, will change it to pass an optional config object into the method